Modules/BenchPress.Azure/Classes/AuthenticationResult.psm1

```diff
@@ -1,4 +1,4 @@
-# INLINE_SKIP
+﻿# INLINE_SKIP
```

Comment:
This type of diff means the encoding was changed from UTF-8 to UTF-8 with BOM

Reply:
It doesn't display the BOM, but it's there!